e2e-polybft/framework/test-bridge.go

```diff
@@ -462,8 +463,8 @@ func (t *TestBridge) premineNativeRootToken(tokenConfig *polybft.TokenConfig,
 			"--jsonrpc", t.JSONRPCAddr(),
 			"--premine-amount", premineAmount.String(),
 			"--stake-amount", stakedAmount.String(),
-			"--erc20-token", polybftConfig.Bridge.RootNativeERC20Addr.String(),
-			"--blade-manager", polybftConfig.Bridge.BladeManagerAddr.String(),
+			"--erc20-token", polybftConfig.Bridge[tokenConfig.ChainID].RootNativeERC20Addr.String(),
+			"--blade-manager", polybftConfig.Bridge[tokenConfig.ChainID].BladeManagerAddr.String(),
 		}
```

Comment:
Extract nativeTokenChainCfg to a variable, and reuse it: 
`nativeTokenChainCfg := polybftConfig.Bridge[tokenConfig.ChainID]`
```suggestion
			"--erc20-token", nativeTokenChainCfg.RootNativeERC20Addr.String(),
			"--blade-manager", nativeTokenChainCfg.BladeManagerAddr.String(),
```